lib/btc-utils.js

```diff
@@ -184,7 +186,20 @@ const waitForBitcoinMempoolToGetTxs = async (btcTxHelper, maxAttempts = 3, check
 
   const getBtcAddressBalanceInSatoshis = async (btcTxHelper, btcAddress) => {
     return Number(btcToSatoshis(await btcTxHelper.getAddressBalance(btcAddress)));
-  };  
+  };
+
+  /**
+   * Gets the list of utxos that sum up to the `amountInBtc` for the given btc address, and includes the calculated change.
+   * Once the sum of the selected utxos is equal o greater than the `amountInBtc`, no more utxos will be added.
+   * This doesn't return the list of utxos from the account, only utxos that sum up to `amountInBtc`.
+   * @param {BtcTransactionHelper} btcTxHelper to make calls in the bitcoin network
+   * @param {string} address the btc account address to get the utxos from
+   * @param {number} amountInBtc the amount in btc the utxos need to sum up to
+   * @returns {Promise<import('btc-transaction-helper').SpendableUtxosInformation>} an object containing the list of `utxos` and the `change` difference of the sum of utxos minus the `amountInBtc`
+   */
+  const getAddressUtxosInfo = async (btcTxHelper, address, amountInBtc) => {
```

Comment:
I think this name is not very explanatory, by the name I thought it would return the list of available utxos for a given address.

Alos not sure this function makes much sense, it basically takes the 3 parameters passed and makes the call. What's the difference with calling btcTxHelper directly?

Reply:
Updated.